test/libyul/yulOptimizerTests/rematerialiser/for_continue.yul

```diff
@@ -0,0 +1,36 @@
+{
+	let a
+	let b
+	for {let i := 0} lt(i, 10) {i := add(a, b)}
```

Comment:
Perhaps add a comment: `a` might be reset after the continue, so we can only replace `b` which is known to be `caller()`.